Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -66,6 +67,21 @@ def livecheck_formulae(formulae_to_check, args)
       @livecheck_strategy_names.freeze
 
       has_a_newer_upstream_version = false
+
+      if args.json? && !args.quiet? && $stdout.tty?
+        total_formulae = if formulae_to_check == Formula
+          formulae_to_check.count
+        else
+          formulae_to_check.length
+        end
+        ohai "Running checks"
+        progress = ProgressBar.create(
+          total:          total_formulae,
+          progress_mark:  "#",
+          remainder_mark: ".",
+          format:         " %t: [%B] %c/%C ",
```

Comment:
```suggestion
          format:         " %t: [%B] %c/%C ",
          output:         $stderr,
```

Reply:
Although this solves our problem, would it be _okay_ to write non-error output to `stderr` by default?

Reply:
Yeah, `stderr` is often for diagnostic information, not just errors -- https://unix.stackexchange.com/a/331808.
The curl progress bar is written to `stderr`, for instance.

Reply:
TIL, thank you! The link was very helpful.